templates/microcloud/roadshow.html

```diff
@@ -0,0 +1,268 @@
+          </div>
+        </div>
+      </div>
+      <hr class="p-rule" />
+      <div class="p-block">
+        <div class="row">
+          <div class="col-3">
+            <h3 class="u-sv-1">Munich</h3>
+            <ul class="p-media-object__meta-list">
+              <li class="p-media-object__meta-list-item--date">
+                <span class="u-off-screen">Date: </span>21 November 2023
+              </li>
+              <li class="p-media-object__meta-list-item--location">
+                <span class="u-off-screen">Location: </span><a href="https://goo.gl/maps/hsZnpASmoD7vwXUC6">Marias Platzl, Mariahilfplatz</a>
+              </li>
+            </ul>
+          </div>
+          <div class="col-3 u-vertically-center">
+            <p class="u-no-margin--bottom">
+              <a href="https://ubuntu.com/engage/microcloud-launch-event-munich" class="p-button--positive u-no-margin--bottom">Register here</a></p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row u-no-padding--top">
+    <hr class="p-rule" />
+    <div class="col-6 col-medium-3">
+      <div class="p-block">
+        <h2>MicroCloud resources</h2>
+      </div>
+    </div>
+    <div class="col-6 col-medium-3">
+      <div class="p-block">
+        <p class="p-text--small-caps u-no-margin--bottom">DataSheet</p>
+        <h3 class="p-heading--2 u-no-padding--top"><a
+                href="https://pages.ubuntu.com/rs/066-EOV-335/images/MicroCloud_Datasheet.pdf">MicroCloud
+          DataSheet</a></h3>
+      </div>
+      <hr class="p-rule" />
+      <div class="p-block">
+        <p class="p-text--small-caps u-no-margin--bottom">Whitepaper</p>
+        <h3 class="p-heading--2 u-no-padding--top"><a href="https://ubuntu.com/engage/guide-to-micro-clouds">CTO's
+          guide to MicroCouds</a></h3>
+      </div>
+      <hr class="p-rule" />
+      <div class="p-block">
+        <p class="p-text--small-caps u-no-margin--bottom">Video</p>
+        <h3 class="p-heading--2 u-no-padding--top"><a href="https://www.youtube.com/watch?v=iWZYUU8lX5A">MicroCloud
+          demo</a></h3>
+      </div>
+      <hr class="p-rule" />
+      <div class="p-block">
+        <p class="p-text--small-caps u-no-margin--bottom">Blog</p>
+        <h3 class="p-heading--2 u-no-padding--top"><a
+                href="https://ubuntu.com/blog/minimising-latency-in-your-edge-cloud-with-real-time-kernel">MicroClouds
+          and real-time kernel</a></h3>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--white is-deep">
+  <div class="row">
+    <div class="col-12">
+      <div class="p-block">
+        <p class="p-heading--2 u-no-margin--bottom">
+          Already have a use case in mind? <a href="/contact-us" class="js-invoke-modal">Reach out to us</a> to
```

Comment:
This is a modal in the old page?

Reply:
It has to be updated to the new modal, but I'm handling that in a separate pr. Left a comment for the time being to clarify